_pages/en_US/troubleshooting.txt

```diff
@@ -118,68 +118,90 @@ Python is not installed on the computer you are using. Download it from the [Pyt
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Error 01: Couldn't find Nintendo 3DS folder</u></summary>
+<summary><u>HOME menu extdata: Missing!</u></summary>
 
-You are not running MSET9 from the root of the SD card, or the SD card is missing the Nintendo 3DS folder.
+Please power on your console with your SD inserted, then check the MSET9 status again.
 
-Remember, your SD card should look like this:
+If this does not work, your SD card needs to be formatted:
 
-![MSET9 root layout](/images/screenshots/mset9/mset9-root-layout.png)
-{: .notice--info}
-
-If your SD card layout is correct, then your SD card most likely isn't being read by your console. Re-format it ([Windows](formatting-sd-(windows)), [Linux](formatting-sd-(linux)), [macOS](formatting-sd-(mac))), then start again from the beginning of [Section I](installing-boot9strap-(mset9-cli)#section-i---prep-work).
+1. Copy everything off the SD Card to your PC
+1. Format it ([Windows](formatting-sd-(windows)), [Linux](formatting-sd-(linux)), [macOS](formatting-sd-(mac)))
+1. Copy everything back
+1. Start again from [Section I Step 7](installing-boot9strap-(mset9-cli)#section-i---prep-work).
 
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Error 02: Your SD is write protected</u></summary>
+<summary><u>Mii Maker extdata: Missing!</u></summary>
 
-Write-protection is enabled on this SD card. If you are using a full-size SD card, ensure that the lock is flipped in the [upright position](/images/sdlock.png). Otherwise, try ejecting and reinserting your SD card.
+Mii Maker data was not found on the SD card. Please power on your console with your SD inserted, then launch Mii Maker, then check the MSET9 status again.
+
+If this does not work, your SD card needs to be formatted:
+
+1. Copy everything off the SD Card to your PC
+1. Format it ([Windows](formatting-sd-(windows)), [Linux](formatting-sd-(linux)), [macOS](formatting-sd-(mac)))
+1. Copy everything back
+1. Start again from [Section I Step 7](installing-boot9strap-(mset9-cli)#section-i---prep-work).
 
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Error 03: Don't change console version</u></summary>
+<summary><u>Title database: Not initialized!</u></summary>
+
+Ensure you have reset the databases. Please power on your console with your SD inserted then launch System Settings and navigate to `Data Management` -> `Nintendo 3DS` -> `Software` and press Reset ([image](/images/screenshots/database-reset.jpg))
+    + This will not actually wipe your data
+If you get a reset prompt, after resetting, power off your 3DS and start again from [Section I Step 13](installing-boot9strap-(mset9-cli)#section-i---prep-work).
```

Comment:
this should also probably be separated into bullet points as it is difficult to parse